### PR TITLE
Update livewordcloud.py

### PR DESCRIPTION
**Pull request by LianeHughes**
Added stopwords (words to be exluded) and word combinations (words to be merged and counted together).

### CODE REVIEW

File: livewordcloud.py [hunk 1]
```diff
@@ -25,12 +25,17 @@ def gen_wordcloud(field="title"):
     resp = requests.get("https://publications-covid19.scilifelab.se/publications.json")
     txt = resp.json()
 
-    # the below level of normalisation will give access to abstract and the title -
+    # normalisation at this level accesses title/abstract
     # authors requires further 'digging' in the .json
     df = pd.json_normalize(txt["publications"])
+    df.replace("antibody", "antibodies",regex=True, inplace=True)
```
Comment (talavis) [inline livewordcloud.py]: Minor space missing here: `,regex=True,`

File: livewordcloud.py [hunk 2]
```diff
@@ -80,7 +85,7 @@ def multi_color_func(word=None,
 
     img = io.BytesIO()
 
-    # savefig will save the figure (at resolution 300dpi - good enoough for print)
+    # savefig will save the figure (resolution 300dpi - good enoough for print)
```
Comment (talavis) [inline livewordcloud.py]: Funny that you removed "at" but kept "enoough" :)

**Review by talavis — CHANGES_REQUESTED** [review]

File: livewordcloud.py (hunk 1 above)
Comment (LianeHughes) [reply livewordcloud.py]: I added it :) thanks

File: livewordcloud.py (hunk 2 above)
Comment (LianeHughes) [reply livewordcloud.py]: haha I was looking to drop 2 characters and I saw 'at' :D

**Review by talavis — APPROVED** [review]

**Pull request merged**